### PR TITLE
#166272327 Deploy app to Heroku

### PR DESCRIPTION
**Pull request by EMacco**
#### What does this PR do?
Finish up app deployment to Heroku

#### Description of Task to be completed?
- Add sequelize-cli as a dependency
- Add script to run migration after the app has been built
- Set env variables in app.json
- Set addons in app.json

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166272327

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Review by adex001 — COMMENTED** [review]
Looks good to me. You need to resolve merge conflict.

File: app.json [hunk 1]
```diff
@@ -2,11 +2,20 @@
   "name": "kingsmen-ah-backend",
   "description": "A community of like minded authors that fosters inspiration and innovation by leveraging the modern web.",
   "scripts": {},
```
Comment (fidelisojeah) [inline app.json]: include the following script 
    `"postdeploy": "heroku config:set HOST_NAME=${HEROKU_APP_NAME}.herokuapp.com --app ${HEROKU_APP_NAME} && npm run refreshdb"`
 we would look more into it later

File: app.json [hunk 2]
```diff
@@ -2,11 +2,20 @@
...
-  "env": {},
+  "env": {
+    "SALT": {
```
Comment (fidelisojeah) [inline app.json:8]: include the following env
```json
"HEROKU_APP_NAME": {
      "required": true
    },
    "HEROKU_PARENT_APP_NAME": {
      "required": true
    }
```

File: app.json (hunk 1 above)
Comment (EMacco) [reply app.json]: Ok Thanks

File: app.json (hunk 2 above)
Comment (EMacco) [reply app.json]: Will do. I am guessing I will have to set those in Heroku environment too?

Comment (EMacco) [reply app.json:8]: Ok I understand now. Thanks

**Review by adex001 — APPROVED** [review]

**Review by despeauxz — APPROVED** [review]
Everything looks good. kudos man

**Pull request merged**